llvm/include/llvm/MCCAS/MCCASObjectV1.h

```diff
@@ -329,7 +329,7 @@ class SpecificRef : public MCObjectProxy {
     static constexpr StringLiteral KindString = #MCEnumIdentifier;             \
     static Expected<MCFragmentName##Ref>                                       \
     create(MCCASBuilder &MB, const MCFragmentName &Fragment,                   \
-           unsigned FragmentSize);                                             \
+           unsigned FragmentSize, ArrayRef<char> FragmentContents);            \
```

Comment:
Is `FragmentSize != FragmentContents.size()`?
That's what the current interface says is possible, but I'm not sure why that could be the case

Reply:
The `FragmentSize` is the size of the fragment before splitting, the `FragmentContents` are dedup-able contents of the fragment, we want to pass the original `FragmentSize` to make sure any fragment padding is the same as it would have been if we didn't split the fragment contents